docs/how-to/how-to-run-docker-with-kata.md

```diff
@@ -0,0 +1,128 @@
+# How to run Docker in Docker with Kata Containers
+
+This document describes the why and how behind running Docker in a Kata Container.
+
+> **Note:** While in other environments this might be described as "Docker in Docker", Kata 2.x *cannot* be run with Docker.
+
+## Requirements
+
+- A working Kata Containers installation
+
+## Install and configure Kata Containers
+
+Follow the [Kata Containers installation guide](../install/README.md) to Install Kata Containers on your Kubernetes cluster.
+
+## Background
+
+Docker in Docker ("DinD") is the colloquial name for the ability to run `docker` from inside a container.
+
+You can learn more about about Docker-in-Docker at the following links:
+
+- [The original announcement of DinD](https://www.docker.com/blog/docker-can-now-run-within-docker/)
+- [`docker` image Docker Hub page](https://hub.docker.com/_/docker/) (this page lists the `-dind` releases)
+
+While normally DinD refers to running `docker` from inside a Docker container, Kata Containers 2.x allows only supported runtimes (such as [`containerd`](../install/container-manager/containerd/containerd-install.md)).
+Running `docker` in a Kata Container implies creating Docker containers from inside a container managed by `containerd` (or another supported container manager).
```

Comment:
Optional but I do wonder whether a picture could help explain this further. If you do want to add one, ASCII art in ` ``` ` is perfectly acceptable (for one example (there are others) see https://github.com/kata-containers/kata-containers/blob/main/docs/tracing.md#agent-tracing-architecture). If you want to use another graphics format, please see:

- https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#images.

But even something like this might help:

```
container manager -> Kata Containers shim     -> Docker Daemon -> Docker container
(containerd)        (containerd-shim-kata-v2)    (dockerd)        (busybox sh)
```

Or even something like this:

| Level description | Host workload | Kata VM workload |
|-|-|-|
| Container manager | `containerd` | |
| Kata Containers | `containerd-shim-kata-v2` | Docker (`dockerd`) |
| Docker container | | `busybox sh` |

Reply:
I think that textual description you have works great -- I'll use that!